M2/Macaulay2/m2/startup.m2.in

```diff
@@ -609,10 +609,6 @@ loadCore = path -> (
 
 (loadDepth = 3; processCommandLineOptions 3; loadDepth = 0)
 
-if class Core =!= Symbol and not core "noinitfile" then (
-    -- the location of init.m2 is documented in the node "initialization file"
```

Comment:
Would be good to keep this comment in `last.m2`.

Also, I know this wasn't possible before either, but is there a scenario in which we might want Core not loaded but `init.m2` loaded?

Reply:
Sounds good -- I'll restore the comment.

I suppose if the user runs `M2` with `--no-core` but not `-q` then `init.m2` should be loaded.

Reply:
I doubt this would function, unless if `init.m2` loads another version of Core?

Reply:
I've added the comment back.

I realized that if we don't load Core, then we won't have added any of the start functions, and so with this change we wouldn't load init.m2.  I suppose it would be easy to add a line that loads `init.m2` in `startup.m2.in` if `nocore` is true, but I'm not sure how useful it would be.